base/metrics/field_trial.cc

```diff
@@ -192,7 +192,10 @@ void OnOutOfMemory(size_t size) {
 #if BUILDFLAG(IS_NACL)
   NOTREACHED();
 #else
+// TODO(b/382115610): Re-enable use_partition_alloc
+#if !BUILDFLAG(IS_STARBOARD)
```

Comment:
Is this what C25 does?

Reply:
#if BUILDFLAG(IS_NACL) || defined(STARBOARD)
  NOTREACHED();
#else


ahh C25 does this ^ . I'll copy over the same idiom

Reply:
done,copied over from 25lts